python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -1237,13 +1237,23 @@ def test_sql(self):
         self.assertEqual(1, len(pdf.index))
 
     def test_sql_with_named_args(self):
-        df = self.connect.sql("SELECT * FROM range(10) WHERE id > :minId", args={"minId": 7})
-        df2 = self.spark.sql("SELECT * FROM range(10) WHERE id > :minId", args={"minId": 7})
+        from pyspark.sql.functions import create_map, lit
+        from pyspark.sql.connect.functions import lit as clit
+        from pyspark.sql.connect.functions import create_map as ccreate_map
```

Comment:
https://github.com/apache/spark/blob/8c27de68756d4b0e5940211340a0b323d808aead/python/pyspark/sql/tests/connect/test_connect_basic.py#L78-L79

I think you can use already imported `SF` and `CF`, to be consistent with other tests

Reply:
@zhengruifeng just wondering should `SF` and `CF` be capitalized here?

Reply:
@allisonwang-db since they are just used in tests, so I guess doesn't matter.

for new test file, I think we can start with `sf` and `cf`.